### ISSUE TITLE
Adding support to Swift Package Manager

### ISSUE THREAD

**Issue by rextremotabaresDtt** [Pushwoosh/pushwoosh-ios-sdk#91]
With XCode 12 binary frameworks can be embedded in packages distributed with SPM (https://developer.apple.com/videos/play/wwdc2020/10147/) . 
Could you please consider adding support for Swift Package Manager?

**Comment by gameaccount** [comment]
Hi @wfhm, is this something you guys are working on and if so, is there a time frame?

**Comment by wfhm** [comment]
@gameaccount,

We will definitely do it in future, however, right now we do not have any ETA on it, so it will most likely not be added in 2020. I will update this thread when there is any news on the ETA.

**Comment by gameaccount** [comment]
@wfhm, hmm okay, instead of that then would it be possible to just provide the framework binaries (https://github.com/Pushwoosh/pushwoosh-ios-sdk/tree/master/Framework) as .xcframework files? If not we have no way of adding them to our Swift packages.

**Comment by wfhm** [comment]
@gameaccount,

We are not going to release .xcframework files publicly until SPM is officially supported. As a temporary solution you can try the ones attached to this response.

**Comment by dungi** [comment]
I would also like to see spm support for out project

**Comment by dungi** [comment]
Is this xcframework always up to date?

**Comment by wfhm** [comment]
@dungi no, it was never updated since it was posted. Below are the new versions with the latest Pushwoosh iOS SDK:

[PushwooshInboxUi.xcframework.zip](https://github.com/Pushwoosh/pushwoosh-ios-sdk/files/6396224/PushwooshInboxUi.xcframework.zip)
[Pushwoosh.xcframework.zip](https://github.com/Pushwoosh/pushwoosh-ios-sdk/files/6396221/Pushwoosh.xcframework.zip)

**Comment by dungi** [comment]
I guess the xcframework was build wrong. If I run it on real device it works. But for simulator the following error

symbol(s) not found for architecture arm64

**Comment by wfhm** [comment]
@dungi sorry, you are right. Please try these ones:

[Pushwoosh.xcframework.zip](https://github.com/Pushwoosh/pushwoosh-ios-sdk/files/6427301/Pushwoosh.xcframework.zip)
[PushwooshInboxUI.xcframework.zip](https://github.com/Pushwoosh/pushwoosh-ios-sdk/files/6427307/PushwooshInboxUI.xcframework.zip)

**Comment by murillotomas** [comment]
Hello, is there any update on this issue? Pushwoosh is the last remaining dependency in my project I have to be fully on Swift Package Manager.
Thanks.

**Comment by dungi** [comment]
@wfhm - can you provide a new xcframework for the lastest version?

**Comment by rextremotabaresDtt** [comment]
Any update on this?

**Comment by wfhm** [comment]
@rextremotabaresDtt 

We are still working on supporting SPM, I will check with the team and provide an ETA once there is any. Meanwhile, as a temporary workaround, .xcframework files are available for all new releases already - you can find them the Pushwoosh.framework.zip in assets for every release:

https://github.com/Pushwoosh/pushwoosh-ios-sdk/releases/tag/6.2.5
